File: core-sample-app/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/core/sampleapp/utils/VideoIdsProvider.java
package com.pierfrancescosoffritti.androidyoutubeplayer.core.sampleapp.utils;

import java.util.Random;

public class VideoIdsProvider {
  private static final String[] videoIds = {"-e_3Cg9GZFU", "LvetJ9U_tVY", "S0Q4gqBUs7c", "kqSdQq5bklE", "n365C9NbbC4"};
  private static final String[] liveVideoIds = {"hHW1oY26kxQ"};
  private static final Random random = new Random();

  public static String getNextVideoId() {
    return videoIds[random.nextInt(videoIds.length)];
  }

  public static String getNextLiveVideoId() {
    return liveVideoIds[random.nextInt(liveVideoIds.length)];
  }
}

File: chromecast-sender-sample-app/src/main/java/com/pierfrancescosoffritti/androidyoutubeplayer/chromecast/sampleapp/utils/VideoIdsProvider.kt
package com.pierfrancescosoffritti.androidyoutubeplayer.chromecast.sampleapp.utils

import java.util.*

object VideoIdsProvider {
    private val videoIds = arrayOf("6JYIGclVQdw", "LvetJ9U_tVY", "S0Q4gqBUs7c", "zOa-rSM4nms")
    private val random = Random()

    fun getNextVideoId(): String {
        return videoIds[random.nextInt(videoIds.size)]
    }
}